### PR TITLE
Qualcomm CVEs for October.

### PR DESCRIPTION
**Pull request by QC-CNA**

### CODE REVIEW

File: 2022/25xxx/CVE-2022-25666.json [hunk 1]
```diff
@@ -1,18 +1,69 @@
...
...065, IPQ8068, IPQ8070, IPQ8070A, IPQ8071A, IPQ8072A, IPQ8074A, IPQ8076, IPQ8076A, IPQ8078, IPQ8078A, IPQ8173, IPQ8174, MDM9150, MDM9650, MSM8996AU, PMP8074, QAM8295P, QCA4024, QCA6310, QCA6320, QCA6335, QCA6390, QCA6391, QCA6420, QCA6426, QCA6430, QCA6436, QCA6564, QCA6564A, QCA6564AU, QCA6574, QCA6574A, QCA6574AU, QCA6595AU, QCA6696, QCA7500, QCA8072, QCA8075, QCA8081, QCA8337, QCA9880, QCA9886, QCA9888, QCA9889, QCA9898, QCA9980, QCA9984, QCA9985, QCA9990, QCA9992, QCA9994, QCC5100, QCN5021, QCN5022, QCN5024, QCN5052, QCN5054, QCN5122, QCN5124, QCN5152, QCN5154, QCN5164, QCN6023, QCN6024, QCN6122, QCN6132, QCN9000, QCN9012, QCN9022, QCN9024, QCN9070, QCN9072, QCN9074, QCN9100, QCS410, QCS610, QCS8155, QSM8250, Qualcomm215, SA515M, SA6145P, SA6150P, SA6155, SA6155P, SA8145P, SA8150P, SA8155, SA8155P, SA8195P, SA8295P, SD 8 Gen1 5G, SD205, SD210, SD429, SD835, SD845, SD855, SD865 5G, SD870, SD888 5G, SDA429W, SDM429W, SDX55, SDX55M, SDXR2 5G, SW5100, SW5100P, WCD9335, WCD9340, WCD9341, WCD9370, WCD9380, WCD9385, WCN3610, WCN3620, WCN3660B, WCN3680B, WCN3950, WCN3980, WCN3988, WCN3990, WCN3998, WCN6850, WCN6851, WCN6855, WCN6856, WCN7850, WCN7851, WSA8810, WSA8815, WSA8830, WSA8835"
```
Comment (anthnysingleton) [inline 2022/25xxx/CVE-2022-25666.json]: Your list of product versions is too long here exceeding 1300 chars. Please parse your product_data into multiple arrays with the version value not exceeding 1300 chars.

```
"product_data":[
     {
        "product_name": "Product 1",
        "version": {
             "version_data": [
                   {
                       "version_value": "affected/fixed versions for Product 1"
                   }
                ]
         }
     },
    {
        "product_name": "Product N",
        "version": {
             "version_data": [
                   {
                       "version_value": "affected/fixed versions for Product N"
                   }
                ]
         }
     }
]
```

File: 2022/25xxx/CVE-2022-25687.json [hunk 2]
```diff
@@ -1,18 +1,69 @@
...
+      "STATE": "PUBLIC"
+   }, 
+   "affects": {
+      "vendor": {
+         "vendor_data": [
             {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+               "product": {
+                  "product_data": [
```
Comment (anthnysingleton) [inline 2022/25xxx/CVE-2022-25687.json]: Your list of product versions is too long here exceeding 1300 chars. Please parse your product_data into multiple arrays with the version value not exceeding 1300 chars.

```
"product_data":[
     {
        "product_name": "Product 1",
        "version": {
             "version_data": [
                   {
                       "version_value": "affected/fixed versions for Product 1"
                   }
                ]
         }
     },
    {
        "product_name": "Product N",
        "version": {
             "version_data": [
                   {
                       "version_value": "affected/fixed versions for Product N"
                   }
                ]
         }
     }
]
```

File: 2022/25xxx/CVE-2022-25718.json [hunk 3]
```diff
@@ -1,18 +1,69 @@
...
+      "STATE": "PUBLIC"
+   }, 
+   "affects": {
+      "vendor": {
+         "vendor_data": [
             {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+               "product": {
+                  "product_data": [
```
Comment (anthnysingleton) [inline 2022/25xxx/CVE-2022-25718.json]: Your list of product versions is too long here exceeding 1300 chars. Please parse your product_data into multiple arrays with the version value not exceeding 1300 chars.

```
"product_data":[
     {
        "product_name": "Product 1",
        "version": {
             "version_data": [
                   {
                       "version_value": "affected/fixed versions for Product 1"
                   }
                ]
         }
     },
    {
        "product_name": "Product N",
        "version": {
             "version_data": [
                   {
                       "version_value": "affected/fixed versions for Product N"
                   }
                ]
         }
     }
]
```

File: 2022/25xxx/CVE-2022-25719.json [hunk 4]
```diff
@@ -1,18 +1,69 @@
...
+      "STATE": "PUBLIC"
+   }, 
+   "affects": {
+      "vendor": {
+         "vendor_data": [
             {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+               "product": {
+                  "product_data": [
```
Comment (anthnysingleton) [inline 2022/25xxx/CVE-2022-25719.json]: Your list of product versions is too long here exceeding 1300 chars. Please parse your product_data into multiple arrays with the version value not exceeding 1300 chars.

```
"product_data":[
     {
        "product_name": "Product 1",
        "version": {
             "version_data": [
                   {
                       "version_value": "affected/fixed versions for Product 1"
                   }
                ]
         }
     },
    {
        "product_name": "Product N",
        "version": {
             "version_data": [
                   {
                       "version_value": "affected/fixed versions for Product N"
                   }
                ]
         }
     }
]
```

File: 2022/25xxx/CVE-2022-25720.json [hunk 5]
```diff
@@ -1,18 +1,69 @@
...
+      "STATE": "PUBLIC"
+   }, 
+   "affects": {
+      "vendor": {
+         "vendor_data": [
             {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+               "product": {
+                  "product_data": [
```
Comment (anthnysingleton) [inline 2022/25xxx/CVE-2022-25720.json]: Your list of product versions is too long here exceeding 1300 chars. Please parse your product_data into multiple arrays with the version value not exceeding 1300 chars.

```
"product_data":[
     {
        "product_name": "Product 1",
        "version": {
             "version_data": [
                   {
                       "version_value": "affected/fixed versions for Product 1"
                   }
                ]
         }
     },
    {
        "product_name": "Product N",
        "version": {
             "version_data": [
                   {
                       "version_value": "affected/fixed versions for Product N"
                   }
                ]
         }
     }
]
```

File: 2022/25xxx/CVE-2022-25736.json [hunk 6]
```diff
@@ -1,18 +1,69 @@
...
+      "STATE": "PUBLIC"
+   }, 
+   "affects": {
+      "vendor": {
+         "vendor_data": [
             {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+               "product": {
+                  "product_data": [
```
Comment (anthnysingleton) [inline 2022/25xxx/CVE-2022-25736.json]: Your list of product versions is too long here exceeding 1300 chars. Please parse your product_data into multiple arrays with the version value not exceeding 1300 chars.

```
"product_data":[
     {
        "product_name": "Product 1",
        "version": {
             "version_data": [
                   {
                       "version_value": "affected/fixed versions for Product 1"
                   }
                ]
         }
     },
    {
        "product_name": "Product N",
        "version": {
             "version_data": [
                   {
                       "version_value": "affected/fixed versions for Product N"
                   }
                ]
         }
     }
]
```

File: 2022/25xxx/CVE-2022-25748.json [hunk 7]
```diff
@@ -1,18 +1,69 @@
...
+      "STATE": "PUBLIC"
+   }, 
+   "affects": {
+      "vendor": {
+         "vendor_data": [
             {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+               "product": {
+                  "product_data": [
```
Comment (anthnysingleton) [inline 2022/25xxx/CVE-2022-25748.json]: Your list of product versions is too long here exceeding 1300 chars. Please parse your product_data into multiple arrays with the version value not exceeding 1300 chars.

```
"product_data":[
     {
        "product_name": "Product 1",
        "version": {
             "version_data": [
                   {
                       "version_value": "affected/fixed versions for Product 1"
                   }
                ]
         }
     },
    {
        "product_name": "Product N",
        "version": {
             "version_data": [
                   {
                       "version_value": "affected/fixed versions for Product N"
                   }
                ]
         }
     }
]
```

File: 2022/25xxx/CVE-2022-25749.json [hunk 8]
```diff
@@ -1,18 +1,69 @@
...
+      "STATE": "PUBLIC"
+   }, 
+   "affects": {
+      "vendor": {
+         "vendor_data": [
             {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+               "product": {
+                  "product_data": [
```
Comment (anthnysingleton) [inline 2022/25xxx/CVE-2022-25749.json]: Your list of product versions is too long here exceeding 1300 chars. Please parse your product_data into multiple arrays with the version value not exceeding 1300 chars.

```
"product_data":[
     {
        "product_name": "Product 1",
        "version": {
             "version_data": [
                   {
                       "version_value": "affected/fixed versions for Product 1"
                   }
                ]
         }
     },
    {
        "product_name": "Product N",
        "version": {
             "version_data": [
                   {
                       "version_value": "affected/fixed versions for Product N"
                   }
                ]
         }
     }
]
```

**Review by anthnysingleton — CHANGES_REQUESTED** [review]
Hello @QC-CNA,

We have not been able to fully process your PR. There are 8 records that contain "version_value" strings that are too long. Can you please parse out your product version information into separate arrays? I have provided a high level structure in the review. 

Thanks,
CVE Content Team

**Comment by QC-CNA** [comment]
Changes updated on this branch as per recommendation.

**Comment by QC-CNA** [comment]
Hi Team,
Updated the CVE JSON files as per the recommendation. Please let us know if anything else is needed.


Padmalochan Panda
Qualcomm Product Security

**Pull request merged**